llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
Well, even if it returns a copy, it wouldn't necessarily be an immutable list. I'm not sure why one way would be better than another. I would argue returning a copy is better, because the original version cannot be altered by the caller.